### ISSUE TITLE
use_direction_classifier=False causes AttributeError for NoneType

### ISSUE THREAD

**Issue by filaPro** [open-mmlab/mmdetection3d#428]
Looks like `use_direction_classifier=False` bring us [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/dense_heads/anchor3d_head.py#L260) and after that this `None` gets [here](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/detectors/base.py#L202). Where `_loss` must be a `Tensor` not `None`.

**Comment by Tai-Wang** [comment]
Thanks for your information. Actually, we have not tried to remove the direction classifier for long. The problem does exist from the first look. Do you encounter this bug in your experiments?
Maybe a workaround for now will be setting the default dir loss to 0*something. More elegant ways to avoid this should be adopted later. Or do you have any ideas?

**Comment by filaPro** [comment]
Yes, it's just about my own experiments. Your `0*something` can be a solution.
